tests/integrationv2/test_client_authentication.py

```diff
@@ -140,6 +140,8 @@ def test_client_auth_with_s2n_client_no_cert(managed_process, cipher, curve, pro
     server_options.key = Certificates.RSA_2048_SHA256_WILDCARD.key
     server_options.cert = Certificates.RSA_2048_SHA256_WILDCARD.cert
 
+    client_options.extra_flags = get_keyshare_flag(curve)
```

Comment:
If the `-K` flag has to be added to each test, it would fit better in the S2N provider `setup_client` method. You will have access to `self.options.curve` in that method. By adding it there, you won't have to modify every test.

Reply:
Will move it to `S2N setup_client`.